### PR TITLE
replace MapQuest by Stamen terrain

### PR DESCRIPTION
**Pull request by nibecker**
Fixing #3162 
Stamen terrain has been chosen due to its open CC license and clean look

### CODE REVIEW

**Comment by wardi** [comment]
@amercader you'd be the best person to look at this when you're back

File: ckanext/reclineview/theme/public/vendor/recline/recline.js [hunk 1]
```diff
@@ -2276,16 +2276,16 @@ my.Map = Backbone.View.extend({
...
   _setupMap: function(){
     var self = this;
     this.map = new L.Map(this.$map.get(0));
 
-    var mapUrl = "//otile{s}-s.mqcdn.com/tiles/1.0.0/osm/{z}/{x}/{y}.png";
-    var osmAttribution = 'Map data &copy; 2011 OpenStreetMap contributors, Tiles Courtesy of <a href="http://www.mapquest.com/" target="_blank">MapQuest</a> <img src="//developer.mapquest.com/content/osm/mq_logo.png">';
-    var bg = new L.TileLayer(mapUrl, {maxZoom: 18, attribution: osmAttribution ,subdomains: '1234'});
+    var mapUrl = "//stamen-tiles-{s}.a.ssl.fastly.net/terrain/{z}/{x}/{y}.jpg";
+    var osmAttribution = 'Map tiles by <a href="http://stamen.com">Stamen Design</a>, under <a href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. Data by <a href="http://openstreetmap.org">OpenStreetMap</a>, under <a href="http://creativecommons.org/licenses/by-sa/3.0">CC BY SA</a>.';
+    var bg = new L.TileLayer(mapUrl, {maxZoom: 18, attribution: osmAttribution});
```
Comment (arusahni) [inline ckanext/reclineview/theme/public/vendor/recline/recline.js]: IIRC, the stamen project supports subdomains `a-d`, while leaflet defaults to `a-c`. Should we define the full range?

**Comment by mattfullerton** [comment]
We should not be modifying vendor scripts in CKAN codebase, surely? This is already fixed in recline and needs someone to merge it in, which is a more involved task (I seem to remember the main problem is the Data Explorer buttons looking a bit weird...)

**Comment by amercader** [comment]
This has been backported to 2.3.5, 2.4.3 and 2.5.3. Thanks @nibecker for the patch

**Pull request closed without merging**